applications/RomApplication/tests/thermal_dynamic_test_files/MainKratosROM.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+import KratosMultiphysics.ConvectionDiffusionApplication
+import KratosMultiphysics.RomApplication as romapp
+
+from KratosMultiphysics.assign_scalar_variable_process import AssignScalarVariableProcess
+from KratosMultiphysics.ConvectionDiffusionApplication.apply_thermal_face_process import ApplyThermalFaceProcess
+from KratosMultiphysics.ConvectionDiffusionApplication.convection_diffusion_analysis import ConvectionDiffusionAnalysis
+from KratosMultiphysics.RomApplication import convection_diffusion_transient_rom_solver
+
+import numpy as np
+import json
+import sys
+
+class ConvDiffDynamicROM(ConvectionDiffusionAnalysis):
+
+    def __init__(self,model,project_parameters):        
+        super(ConvDiffDynamicROM,self).__init__(model,project_parameters)
+        self.selected_time_step_solution_container = []
+
+    def _CreateSolver(self):    
+        return convection_diffusion_transient_rom_solver.ROMSolver(self.model, self.project_parameters["solver_settings"])
+        
+    def ModifyInitialGeometry(self):
+        """Here is the place where the BASIS_ROM and the AUX_ID are imposed to each node"""
+        super(ConvDiffDynamicROM,self).ModifyInitialGeometry()

```

Comment:
aren't you defining this Analysis in the python scripts directory? if so please use it